test.c

```diff
@@ -0,0 +1,155 @@
+/*
+ * test.c
+ *
+ *  Created on: Feb 20, 2020
+ *      Author: Youssef Harby
+ */
+
+
+#include "DIO_TS/DIO.h"
+#include <stdio.h>
+
+DIO_Cfg_s cfg_test;
+
+void DIO_initTesting(uint8_t gpio, uint8_t bit, uint8_t dir, uint8_t expected_output)
+{
+	static int initTestCounter = 1;
+	uint8_t output;
+
+	cfg_test.GPIO = gpio;
+	cfg_test.pins = bit;
+	cfg_test.dir = dir;
+
+	output = DIO_init(&cfg_test);
+
+	if(output == expected_output)
+	{
+		printf("DIO_init test %d Passed\n", initTestCounter);

```

Comment:
You can make it more useful by printing:
Test case Number, test data, expected results, actual results, and pass/fail.